src/azure-cli/azure/cli/command_modules/cognitiveservices/tests/latest/test_identity.py

```diff
@@ -29,13 +29,9 @@ def test_cognitiveservices_identity_assign_when_create(self, resource_group):
                  checks=[self.check('name', '{sname}'),
                          self.check('location', '{location}'),
                          self.check('sku.name', '{sku}'),
-                         self.check('properties.provisioningState', 'Creating')])
+                         self.check('properties.provisioningState', 'Succeeded')])
 
-        for i in range(10):
-            time.sleep(15)
-            account = self.cmd('az cognitiveservices account show -n {sname} -g {rg}').get_output_in_json()
-            if 'Creating' != account['properties']['provisioningState']:
-                break
```

Comment:
There is no need to monitor the status here, because the SDK of Track 2 already waits for the creation operation to complete, right?

Reply:
Yes, so this one is unnecessary.

Reply:
Got it, thanks~